fuels-contract/src/script.rs

```diff
@@ -25,11 +25,14 @@ impl Script {
     }
 
     pub async fn call(self, fuel_client: &FuelClient) -> Result<Vec<Receipt>, Error> {
-        let tx_id = fuel_client.submit(&self.tx).await.unwrap();
+        let tx_id = fuel_client.submit(&self.tx).await.unwrap().0.to_string();
 
-        let receipts = fuel_client.receipts(&tx_id.0.to_string()).await.unwrap();
-
-        Ok(receipts)
+        let receipts = fuel_client.receipts(&tx_id).await.unwrap();
+        let status = fuel_client.transaction_status(&tx_id).await.unwrap();
```

Comment:
Since we're already making some improvements to the error handling in this section, can we get rid of these `unwrap()`s and do some neat error conversions so that we return an `Error` here instead of panicking?